dll/win32/shell32/utils.cpp

```diff
@@ -762,6 +762,133 @@ SHCreatePropertyBag(_In_ REFIID riid, _Out_ void **ppvObj)
     return SHCreatePropertyBagOnMemory(STGM_READWRITE, riid, ppvObj);
 }
 
+// The helper function for SHGetUnreadMailCountW
+static DWORD
+SHELL_ReadSingleUnreadMailCount(
+    _In_ HKEY hKey,
+    _Out_opt_ PDWORD pdwCount,
+    _Out_opt_ PFILETIME pFileTime,
+    _Out_writes_opt_(cchShellExecuteCommand) LPWSTR pszShellExecuteCommand,
+    _In_ INT cchShellExecuteCommand)
+{
+    DWORD dwType, dwCount, cbCount = sizeof(dwCount);
```

Comment:
You could rename cbCount to something more general (like `cbSize`), so that it can also be used for the other SHQueryValueExW calls below.

Reply:
Used `cbSize`.